scripts/zones/Yhoator_Jungle/Zone.lua

```diff
@@ -33,9 +33,10 @@ zoneObject.onInitialize = function(zone)
 
     xi.conq.setRegionalConquestOverseers(zone:getRegionID())
 
-    xi.helm.initZone(zone, xi.helm.type.HARVESTING)
-    xi.helm.initZone(zone, xi.helm.type.LOGGING)
     xi.chocobo.initZone(zone)
+    xi.helm.initZone(zone, xi.helm.type.LOGGING)
+    xi.helm.initZone(zone, xi.helm.type.HARVESTING)
+    updateRainHarvesting(xi.status.DISAPPEAR)
```

Comment:
Function needs to be defined before this point I think

Reply:
It doesn't. I put them together to keep changes clean, but happy to move it above the init function if the preferred style. Let me know.

Reply:
They're referring to the reason why the CI checks are failing. The updateRainHarvesting should be at the top of the file in order for that to pass since it doesn't know what the function is by this point in this file.